### DISCUSSION TITLE
Add NLP support

### DISCUSSION THREAD

**Discussion by nidhaloff** [nidhaloff/igel#82] [Ideas]
We want to add NLP support.
I heard good things about the [NLTK library](https://www.nltk.org/), maybe we can use it. 

It would be great if someone is already familiar with it and want to start working on it ;)

I don't which models are supported by NLTK. This discussion can overlap with the [RNN discussion](https://github.com/nidhaloff/igel/discussions/80)

**Comment by anjali-rgpt** [comment]
What kind of NLP support would you be looking for, exactly? NLTK essentially has a lot of tokenization options and some sample corpora to be used. It can also be used for part of speech tagging. So it's more like having functionalities to help existing models by working on language, than actual models

**Reply by nidhaloff** [reply]
@anjali-rgpt Generally, it would be great to train neural networks for NLP tasks using igel, (for example sentimental analysis). I'm not familiar with NLTK, I just heard good things about it. So the goal is to be able to train neural networks for NLP tasks, that's why I linked the RNN discussion in the description. This may also need some preprocessing (and maybe this is a use case for NLTK).
